roles/containerd/handlers/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+
+- name: Systemctl restart # systemd loads service
+  service:
+    name: "systemd-modules-load.service"
+    state: "restarted"
+  changed_when: false
+  failed_when: false
+
+- name: Sysctl reload  # no reboot
+  command: sysctl --system
+  changed_when: false
+  failed_when: false
+
+- name: restart containerd
+  command: /bin/true
+  notify:
+    - Containerd | restart containerd
+    - Containerd | wait for containerd
```

Comment:
is this a "hack" to trigger always the restart?

Reply:
this is a way to notify two handlers instead of one, in a single `restart containerd` notify action